src/app/api/handle-chapter-request/route.ts

```diff
@@ -67,6 +70,52 @@ export const POST = async (request: NextRequest) => {
             approved: "APPROVED",
           },
         });
+
+        const baseFolder = process.env.GOOGLE_BASEFOLDER; // TODO: make env variable
```

Comment:
We can add `GOOGLE_BASEFOLDER` to `src/env/schema.mjs` to quickly detect missing variables. You can use the environment variables by importing from `src/env/server.mjs` (see `next.config.mjs` for examples)